tests/tests/test-eth-fee/test-eth-txn-weights.ts

```diff
@@ -48,6 +48,21 @@ describeDevMoonbeam("Ethereum Weight Accounting", (context) => {
     let blockWeightsUsed = await apiAt.query.system.blockWeight();
     let normalWeight = blockWeightsUsed.normal.refTime.toBigInt();
     expect(normalWeight).to.equal(EXPECTED_WEIGHT);
+
+    // look for the event for our eth txn
+    let extSuccessEvent = null;
+    let wholeBlock = await context.polkadotApi.rpc.chain.getBlock(block.hash);
+    wholeBlock.block.extrinsics.forEach((ext, index) => {
+      if (ext.method.method == "transact" && ext.method.section == "ethereum") {
+        extSuccessEvent = result.events.filter(
+          ({ event }) => event.method == "ExtrinsicSuccess" && event.section == "system"
```

Comment:
polkadot.js has a better of doing this, via the `api.events....is()` method e.g.:
```
context.polkadotApi.events.ethereum.Executed.is(event)
```

or 
```
context.polkadotApi.events.system.ExtrinsicSuccess.is(event)
```

Reply:
Fixed in 1a9ca2f